app/javascript/components/Results/BestBets/GraphicsBestBet.tsx

```diff
@@ -49,14 +55,27 @@ export const GraphicsBestBet = ({ title, titleUrl, imageUrl, imageAltText, links
         <Grid col={true} className='result-meta-data'>
           <div className='graphics-best-bets-title result-title'>
             {titleUrl ? (
-              <h2 className='result-title-label'><a href={titleUrl} className='result-title-link'>{parse(title)}</a></h2>) : <h2 className='result-title-label'>{parse(title)}</h2>}
+              <h2 className='result-title-label'>
+                <ResultTitle 
+                  url={titleUrl}
+                  clickTracking={() => clickTracking(affiliate, moduleCode.bestBetsGraphics, query, 1, titleUrl, vertical)}>
+                  {parse(title)}
+                </ResultTitle>
+              </h2>) : 
+              <h2 className='result-title-label'>
+                {parse(title)}
+              </h2>}
           </div>
           {links && links.length > 0 && (
             <Grid row gap="md">
               {sortedLinks(links).map((link, index) => {
                 return (
                   <Grid key={index} mobileLg={{ col: 6 }} className='graphics-best-bets-link-wrapper'>
-                    <a href={link.url}>{parse(link.title)}</a>
+                    <ResultTitle 
```

Comment:
@lorawoodford 
This can be like below with passing no class as `<a href={link.url}>{parse(link.title)}</a>` has no class, so this will not break the UI
```
<ResultTitle 
 url={link.url}
clickTracking={() => clickTracking(affiliate, moduleCode.bestBetsGraphics, query, titleUrl ? index + 2 : index + 1, link.url, vertical)}
>
```
Refer implementation at https://github.com/GSA/search-gov/pull/1493/files